cmd/report.go

```diff
@@ -56,6 +74,7 @@ func reportRun(cmd *cobra.Command, args []string) error {
 // nolint // needed for cobra
 func init() {
 	reportCmd.Flags().BoolVarP(&ignoreArchived, "ignore-archived", "i", false, "Ignore archived repositores")
+	reportCmd.Flags().StringVarP(&filePath, "file-path", "f", "report.csv", "file path for report to be created, must be .csv")
```

Comment:
I would use `--output`, I feel like that is "industry standard" for this, but to be honest it does not matter.

Reply:
Already started plumbing it in everywhere, if you're happy can we stick to `file-path` pls